midscribe-core/src/main/resources/css/style.css

```diff
@@ -0,0 +1,552 @@
+/*! Asciidoctor default stylesheet | MIT License | https://asciidoctor.org */
```

Comment:
is default css needed in this project?

Reply:
No it is not needed, the default one from asciidoctor will be used. I removed it from there.